Docs/Configuration.tex

```diff
@@ -8106,6 +8106,19 @@ \subsection{Output Properties}\label{uefioutputprops}
 
 \begin{enumerate}
 
+\item
+  \texttt{InitialMode}\\
+  \textbf{Type}: \texttt{plist\ string}\\
+  \textbf{Failsafe}: \texttt{Auto}\\
+  \textbf{Description}: Selects initial \texttt{ConsoleControl} mode to use before applying \texttt{TextRenderer} option.
+
+  Available values are \texttt{Auto}, \texttt{Text} and \texttt{Graphics}.
```

Comment:
In my opinion, focus on what the mode is in the first place. Something like `In text mode both text and graphics can be rendered onscreen. In graphics mode only graphics can be. Not all tools can switch to text mode if they want to print text.`